scripts/src/main/resources/scripts/command/eclipse

```diff
@@ -363,10 +397,10 @@ function doLoadPluginProperties() {
   then
     file="${SETTINGS_PATH}/eclipse/plugins/${1}.properties"
   fi
-  if [ -f  "${file}" ]
+  if [ -f "${file}" ] 
   then
     plugin_id=""
-    plugin_active="true"
+    plugin_active=""
```

Comment:
It may be a good default to leave this undefined. However, with this change we are changing existing behavior. Do we really want to do this?

Reply:
This is also a documented feature:
https://github.com/devonfw/ide/blob/master/documentation/eclipse.asciidoc#plugins

So changing this and leaving the existing documentation untouched will cause an inconsistency.
I recommend that you revert this change.

Reply:
I don't understand, why this changes existing behaviour, the plugin_active variable gets overridden 3 lines later anyway. (doLoadProperties "${file}")
so if the plugin was true in the settings folder(which should be the default case), then the plugin_active variable will be true aswell.
Nevertheless I changed it to true in the following commit.

Reply:
> I don't understand, why this changes existing behaviour, the plugin_active variable gets overridden 3 lines later anyway. (doLoadProperties "${file}")

That only happens if the *.properties File contains a line saying something about `plugin_active`. Otherwise the defaults defined here in the code apply.